ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -3366,6 +3366,16 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-upi-platform-external
+- as: vsphere-8-upi-ccm-external-f14
+  cron: 38 13 3,18 * *
+  steps:
+    cluster_profile: vsphere-8
```

Comment:
```suggestion
    cluster_profile: vsphere-connected
```

Reply:
I'm confused because this is copied from https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml#L3056-L3065, and I reuse `cucushift-installer-rehearse-vsphere-upi` workflow, Can you please help to explain this?

Reply:
`vsphere-8` jobs is used to cover vsphere 8 platform on purpose, your testing seem like not relevant to vpshere platform version, and `vsphere-connected` is the main cluster profiles that used for QE's testing.

Reply:
Good to know, I am clear now, thank you, will update.